src/scss/components/dropdown-select/mixins/SelectFacade.scss

```diff
@@ -2,40 +2,36 @@
 
 @mixin __selectFacade($focused: false) {
   @if $focused {
-    border: 1px solid $cyan-700;
+    border: 1px solid var(--hxDropdown-focused-borderColor, $cyan-700);
```

Comment:
Please update to the new Color Palette 2.0 color variables.

Reply:
Changes have been done, please review.